ShipperOptions.json

```diff
@@ -2163,31 +2180,45 @@
       },
       {
         "display": "Priority Mail Express Flat Rate Envelope - Post Office To Addressee",
-        "value": "E4"
+        "value": "E4",
+        "weight": false,
```

Comment:
i have no idea if setting the weight on these to false will screw things up, i'm not sure if we updated the dimensions to default to 0,0,0 for these flat rate things or where that would happen.